CHANGELOG.md

```diff
@@ -2,8 +2,9 @@
 
 FEATURES:
 
-* Add support for `recursive` search in `data_vault_namespaces` [#2408](https://github.com/hashicorp/terraform-provider-vault/pull/2408)
+* Add support for `recursive` search in `data_vault_namespaces` [#2408](https://github.com/hashicorp/terraform-provider-vault/pull/2408).
```

Comment:
parenthesis on this line (not around the [#number] ) don't match the next two line - can we please standardize?

Reply:
Fixed.